sdk/boms/azure-sdk-bom/README.md

```diff
@@ -21,7 +21,7 @@ result in all dependencies being included in your project.
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-sdk-bom</artifactId>
-      <version>1.2.19</version>
+      <version>1.2.21</version>
```

Comment:
https://mvnrepository.com/artifact/com.azure/azure-sdk-bom

1.2.20 already released last month but forget to update it in the README